backend/PennCourses/docs_settings.py

```diff
@@ -1012,6 +1016,130 @@ def map_serializer(self, serializer):
 
         return result
 
+    # Overrides, uses overridden method
+    # (https://www.django-rest-framework.org/api-guide/schemas/#map_serializer)
```

Comment:
nit: change this to https://www.django-rest-framework.org/api-guide/schemas/#map_field